docs/wallets/smart-contract-wallets.md

```diff
@@ -0,0 +1,93 @@
+
+Ethereum's account model enables users to deploy and use self-executing smart contracts for a variety of applications. Two types of accounts exist on the Ethereum network: **EOAs** and **smart contract accounts**. Both can send, hold, or receive Ether, but they differ in terms of control and usage. EOAs are controlled by the user through a private key signing function, while smart contract accounts are controlled by their programmed logic.
+
+Standard EOA models have limitations, including the potential for users to **lose or forget their  seed phrase**, **high gas fees** for interacting with smart contracts, and limitations on **wallet customization**. Account abstraction addresses these limitations through 2 approaches: using third-party relayers with an abstracted EOA or using a **layer-2 solution** with native account abstraction. A relayer is an entity that facilitates the transfer of information between different parties on the network, such as between an **L2** and an **L1**, or between one sidechain and another. 
+Safe wallets use the EOA and relayer approach to add customization to the authorization process, while Argent wallets employ [layer-2](scalability/layer-2.md) native account abstraction in partnership with ZKSync.
+
+### What is ERC-4337? 
+
+
+[**ERC-4337**](https://eips.ethereum.org/EIPS/eip-4337) is a proposal to introduce account abstraction to the Ethereum network. It aims to **remove dependence on centralized relayers and EOAs**, offering a more decentralized version of relayer functions. Users can send a series of user operations to a bundler, which differs from a relayer and transaction. However, the current bundler infrastructure is centralized and fragile, raising concerns about its viability.
+Smart contract wallets, enabled by account abstraction, allow for custom signature schemes, such as [**multi-sig**](wallets-multisig.md), **social recovery**, and **quantum-resistant signature** schemes. 
+
+They facilitate multi-operations or atomic operations, batch multiple user operations into a single transaction, and introduce the concept of a paymaster, allowing users to pay gas fees with their preferred token or even off-chain with a credit card.
+
+EIP 4337 proposes a new object called **user operation**. User operations are created, signed by users, and sent to a **user operation mempool**. A **bundler** (a node or network participant that bundles transactions together and submits them to the blockchain for processing) listens to this mempool, picks up new user operations, and bundles them into a **single entity** called a **bundle transaction**, which is then included in an Ethereum block like any other transaction.
+The bundling process involves **two smart contracts**: the wallet smart contract and a special entry point contract. The entry point contract contains most of the logic for user operation validation and execution.
+
+![smart-contract-wallets](ZBF_graphics_May_ERC_4337.jpg)
```

Comment:
`ZBF_graphics_May_ERC_4337` this file hasn't been added

Reply:
Oh yes, because the folder doesn't exist

Reply:
I think I added it?